AdyenComponents/Boleto/BoletoComponentExtensions.swift

```diff
@@ -28,14 +30,17 @@ extension BoletoComponent {
         ///   - localizationParameters: Localization parameters.
         ///   - shopperInformation: Pre-filled optional personal information about the shopper
         ///   - showEmailAddress: Indicates whether to show `sendCopyByEmail` checkbox and email text field
-        public init(style: FormComponentStyle,
+        ///   - payment: The payment information.
+        public init(style: FormComponentStyle = FormComponentStyle(),
+                    payment: Payment? = nil,
```

Comment:
lets not use default nil values, we don't want to encourage  merchant not to provide it :-)